test/puppetlabs/puppetdb/http/facts_test.clj

```diff
@@ -581,74 +580,6 @@
           (is (= body msg))
           (is (= status http/status-bad-request)))))))
 
-(deftestseq ^{:postgres false} two-database-fact-query-config
```

Comment:
@wkalt Do we want to preserve this test? Maybe we could move it to the sync repo where two-databases is already assumed.